server/routes.go

```diff
@@ -0,0 +1,125 @@
+package server
+
+import (
+	"github.com/pkg/errors"
+	"log"
+
+	"github.com/iorhachovyevhen/dsss/models"
+	routing "github.com/qiangxue/fasthttp-routing"
+	"github.com/valyala/fasthttp"
+)
+
+var ErrorBadID = errors.New("wrong id")
+var ErrorBadJSON = errors.New("bad json")
+
+func router() *routing.Router {
+	log.Println("Create router...")
+
+	router := routing.New()
+	router.Group("/file").
+		Post("", addFile).
+		Get(getFile).
+		Delete(deleteFile)
+
+	return router
+}
+
+func addFile(ctx *routing.Context) error {
```

Comment:
All this method can be part of Struct called "Router". Or you can have several Router's and they can add handlers to one global Router, which will add all handlers to server in one place